hawkular-javaagent-wildfly-feature-pack/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml

```diff
@@ -33,9 +33,9 @@ storage-adapter:
   type:      HAWKULAR
   tenant-id: "${hawkular.rest.tenantId:hawkular}"
   feed-id:   "${hawkular.rest.feedId:autogenerate}"
-  url:       "${hawkular.rest.host:http://hawkular-server:8080}"
-  username:  "${hawkular.rest.user}"
-  password:  "${hawkular.rest.password}"
+  url:       "${hawkular.rest.url:http://hawkular-server:8080}"
+  username:  "${env.HAWKULAR_AGENT_USER}"
+  password:  "${env.HAWKULAR_AGENT_PASSWORD}"
```

Comment:
This file does not just affect the agent, are we sure we want to be making this change here?

@jmazzitelli thoughts on this?

Reply:
Hmm.. yeah, this would affect anybody using this file. Like H-Services builds.

Reply:
Isn’t a security issue that they (services and others) are passing this value as a system property? 
If not, could go back to using the ‘sed’

Reply:
@josejulio setting things as environment variables usually isn't a concern (its a common place to put secrets in OpenShift).

Reply:
that's true, too. Maybe we keep this using env. and force people to pass via env vars to avoid putting sysprops on cmd line